spec/features/profile_photo_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+feature "profile photo" do
+
+  let (:user) do
+    User.create(
+    email: "asdf@asdf.com",
```

Comment:
Please indent these as so: 

```
User.create(
  email: "",
  password: ""
)
```

Also, Factories are preferred for test suites.